charts/operator/templates/clusterrole.yaml

```diff
@@ -262,32 +262,3 @@ rules:
       - patch
       - update
 {{- end -}}
-{{- if and .Values.rbac.create ( eq .Values.scope "Namespace" ) -}}
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: {{ include "redpanda-operator.fullname" . }}
-  labels:
-{{ include "redpanda-operator.labels" . | indent 4 }}
-rules:
-  - apiGroups:
-      - ""
-    resources:
-      - nodes
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - persistentvolumes
-    verbs:
-      - delete
-      - get
-      - list
-      - patch
-      - update
-      - watch
-{{- end -}}
```

Comment:
Maybe I'm missing, but will this removal break operator v1 capabilities to decommission Redpandas?

Reply:
This was not supposed to be here, removing it from the commits.